containers/podman-desktop
# feat(extension-api): Implements showInputBox & showQuickPick

### What does this PR do?
Adds new methods `showInputBox` and `showQuickPick`

it allows to interact with the user through extensions 

### Screenshot/screencast of this PR


https://user-images.githubusercontent.com/436777/216383150-9920577b-038d-4b0c-81c5-a9f2b4e8298b.mp4



### What issues does this PR fix or reference?

fixes https://github.com/containers/podman-desktop/issues/1300
fixes https://github.com/containers/podman-desktop/issues/1301


### How to test this PR?

Go to the extensions panel in settings and use as OCI image `quay.io/fbenoit/podman-desktop-quickpick`

you should see items in the status bar, click on it



Change-Id: I769c2491bcd00acc468300a549957b2fd96457c4
Signed-off-by: Florent Benoit <fbenoit@redhat.com>
